### ISSUE TITLE
Cannot add user

### ISSUE THREAD

**Issue by krazyjakee** [sintaxi/surge#380]
Can't add use `surge --add email@domain.com` as it just tries to deploy the project instead of add the user.

**Comment by krazyjakee** [comment]
surely others have this issue? Isn't this a major flaw?

**Comment by rightwork** [comment]
I think this project is on a decline.  I've [seen others](https://github.com/sintaxi/surge/issues/55) report this same issue with no resolution.

In my case I need to change the email on file, but adding a new user, or trying to set up a new user with a new account and following [this guide](https://surge.sh/help/adding-a-custom-domain), does not work.

I'm forced to use and keep around my old email, which is a violation of my new company policy, but it's the only way I can get it to work.

Surge.sh had such a great idea.  I loved the command line abilities to spin up custom domains on the fly - perfect for testing new branches.  It's too bad it didn't take off.  I'll be forced to look for other options.

**Comment by sintaxi** [comment]
@rightwork contact support@surge.sh. This issue tracker is for bug reports and feature requests for the surge CLI.

**Comment by sintaxi** [comment]
@krazyjakee I agree surge needs its own command for inviting users to a project. I will raise priority on this feature.
